### PR TITLE
my pull request

### PR DESCRIPTION
**Pull request by borisworking**

### CODE REVIEW

**Comment by Slooo** [comment]
Yes, is best practice

File: index.html [hunk 1]
```diff
@@ -6,5 +6,6 @@
 	</head>
 
 	<body>
+		The pull request
```
Comment (Slooo) [inline index.html]: amazing text

File: index.html [hunk 2]
```diff
@@ -6,5 +6,6 @@
 	</head>
```
Comment (Slooo) [inline index.html]: why empty string?

Comment (borisworking) [reply index.html]: but a don't know

**Comment by Slooo** [comment]
что такое?

**Pull request merged**